### DISCUSSION TITLE
Issue while creating a service. Shows error 423.

### DISCUSSION THREAD

**Discussion by Prabh74** [emailjs-com/help#115] [Q&A]
While creating a GMAIL service on emailjs, I connected my account and tried to create the service but it shows Error 423, i.e., Account is blocked. I checked the email providers and the provider for my custom email is google only. Any suggestions on what the problem might be?

**Comment by xr0master** [comment]
Go to the Account in the EmailJS dashboard -> Refresh Keys. Then, refresh the page and try to add the service again.
If it doesn't work, you can contact support and ask them why you are blocked. Maybe you have violated the terms of use.
